src/test/kotlin/racingcar/CarsTest.kt

```diff
@@ -2,23 +2,30 @@ package racingcar
 
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
+import racingcar.domain.Car
+import racingcar.domain.CarNamesAndRunCount
+import racingcar.domain.Cars
+import racingcar.domain.NumberCreator
 
 class CarsTest {
-
     @Test
-    fun carCountTest()  {
+    fun carCountTest() {
         val names = "red,blue"
-        val cars = Cars(RacingCarCount(names.split(","),1),AlwaysGoStrategy())
-        val testCarList = listOf(Car("red",1),Car("blue",1))
+        val cars = Cars.runRacingCars(CarNamesAndRunCount(names.split(","), 1), NumberCreator(AlwaysGoStrategy))
+        val testCarList = listOf(Car("red", 1), Car("blue", 1))
         val testCars = Cars(testCarList)
         assertThat(cars).isEqualTo(testCars)
     }
 
     @Test
-    fun createWinningCarListTest()  {
-        val testCarList = listOf(Car("red",2),Car("blue",2),
-            Car("kaki",1))
-        val winningList = listOf(Car("red",2),Car("blue",2))
+    fun createWinningCarListTest() {
+        val testCarList =
+            listOf(
+                Car("red", 2),
+                Car("blue", 2),
+                Car("kaki", 1),
+            )
+        val winningList = listOf(Car("red", 2), Car("blue", 2))
         val cars = Cars(testCarList)
         assertThat(cars.findWinner()).isEqualTo(winningList)
```

Comment:
저는 이러한 시나리오에서 변수명을 지을 때 `expected`, `actual`과 같은 네이밍을 선호하는 편인데요,
```suggestion
        val cars =
            listOf(
                Car("red", 2),
                Car("blue", 2),
                Car("kaki", 1),
            )
        val expected = listOf(Car("red", 2), Car("blue", 2))
        val cars = Cars(testCarList)
        val actual = cars.findWinner()
        assertThat(expected).isEqualTo(actual)
```

AssertJ와 같은 라이브러리에서도 종종 활용되는 네이밍입니다.

https://assertj.github.io/doc

또한 `assertThat` 활용 시 위와 같이 `expected`, `actual`순으로 작성하시면 콘솔에서도 결과값이 의도대로 출력되는 모습을 볼 수 있습니다 🙂 
```
AssertionFailedError: expected: <> but was: <>
```

Reply:
의미적으로 더 명확해보이는 것 같습니다.
수정완료했습니다